javascript/index.js

```diff
@@ -14,52 +14,94 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerText = chronometer.getMinutes()[0];
+  minUni.innerText = chronometer.getMinutes()[1];
+  return chronometer.getMinutes();
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secDec.innerText = chronometer.getSeconds()[0];
+  secUni.innerText = chronometer.getSeconds()[1];
+  return chronometer.getSeconds();
 }
 
 // ==> BONUS
-function printMilliseconds() {
-  // ... your code goes here
-}
+// function printMilliseconds() {
+//   milDec.innerText = chronometer.getMilliseconds()[0];
+//   milUni.innerText = chronometer.getMilliseconds()[1];
+//   return chronometer.getMilliseconds();
+// }
 
 function printSplit() {
-  // ... your code goes here
+  let splitsLi = document.createElement('li');
+  splitsLi.innerHTML = `${printMinutes()}:${printSeconds()}`
+  splitsLi.classList.add('split-time');
+  splits.appendChild(splitsLi);
 }
 
 function clearSplits() {
-  // ... your code goes here
+  let toBeDeletedLi = document.querySelectorAll('.split-time');
+  toBeDeletedLi.forEach((li) => {
+    li.remove()
+  })
```

Comment:
this is a very good approach. There is a simpler one that kinda looks like cheating but writing only below line will actually clear everything inside the splits DOM element, without needing to target the inner elements.
`splits.innerHTML = "";`